privacyidea/static/components/dashboard/views/dashboard.html

```diff
@@ -60,12 +60,16 @@
                                 <td translate>failed users / tokens</td>
                                 <td>
                                     <span ng-repeat="userobj in authentications.users">
-					    {{ ! $first ? "| " : "" }}<a ui-sref="user.details({username: userobj.user, realmname: userobj.realm})">
-                                        {{ userobj.user }}@{{ userobj.realm }}</a>
+                                        {{ !$first ? "| " : "" }}
+                                        <a ui-sref="user.details({
+                                        username: userobj.user,
+                                        realmname: userobj.realm.split(' ')[0]})">
+                                            {{ userobj.user }}@{{ userobj.realm }}
+                                        </a>
```

Comment:
Add here the fails:

~~~~html
 [{{ userobj.fails }}]
~~~~

This will have the same result without misusing the userobj.realm.
~~~~

Reply:
fixed in 2746443f2b3f30f95acb50ab841e51133ace6e9c.